### PR TITLE
Mentioning Laraue.EfCoreTriggers

### PR DESCRIPTION
**Pull request by MaxDZ8**
I am in no way related with the framework nor its author.

This has been just a quick find; I would certainly like someone to evaluate it in more detail. I do not have the time budget to assess it and we could discuss whatever I have the means in the first place.

The repository contains a .dll which is provided to [LoadExtension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlite.sqliteconnection.loadextension?view=msdata-sqlite-6.0.0) for testing. As much as I understand what it _might_ be, it made me raise an eyebrow as I don't enjoy having binaries into repositories.

I am also unable to assess whatever conversion of method calls is reasonably complete.

Nonetheless, I think the API is good, and the library has value.

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -359,3 +359,11 @@ It also has many useful features commonly used especially on web development. Fo
 
 [GitHub repository](https://github.com/SitholeWB/Pagination.EntityFrameworkCore.Extensions)
 [NuGet](https://www.nuget.org/packages/Pagination.EntityFrameworkCore.Extensions)
+
+### Laraue.EfCoreTriggers
+Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
+
+There is some support for method call expression conversion. The library is on par with framework releases and the devoloper seems engaged.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:366]: This seems out of scope for the content here...

File: entity-framework/core/extensions/index.md [hunk 2]
```diff
@@ -359,3 +359,11 @@ It also has many useful features commonly used especially on web development. Fo
 
 [GitHub repository](https://github.com/SitholeWB/Pagination.EntityFrameworkCore.Extensions)
 [NuGet](https://www.nuget.org/packages/Pagination.EntityFrameworkCore.Extensions)
+
+### Laraue.EfCoreTriggers
+Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:364]: Please specify the supported EF Core versions (like it is done for all other extensions in this list)

File: entity-framework/core/extensions/index.md [hunk 3]
```diff
@@ -363,7 +363,7 @@ It also has many useful features commonly used especially on web development. Fo
 ### Laraue.EfCoreTriggers
 Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
 
-There is some support for method call expression conversion. The library is on par with framework releases and the devoloper seems engaged.
+Support for EFCore:  5, 6.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:366]: Please use the same format as everywhere else in this file - so right after the description, add:  `For EF Core: 5, 6.`

File: entity-framework/core/extensions/index.md [hunk 4]
```diff
@@ -361,9 +361,7 @@ It also has many useful features commonly used especially on web development. Fo
 [NuGet](https://www.nuget.org/packages/Pagination.EntityFrameworkCore.Extensions)
 
 ### Laraue.EfCoreTriggers
-Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
-
-Support for EFCore:  5, 6.
+Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite. For EF Core:  5, 6.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:364]: Thanks for ensuring a consistent experience 🙏

**Review by ajcvickers — APPROVED** [review]

File: entity-framework/core/extensions/index.md (hunk 4 above)
Comment (ajcvickers) [reply entity-framework/core/extensions/index.md:364]:
```suggestion

Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite. For EF Core:  5, 6.
```

**Comment by ajcvickers** [comment]
Thanks!
